libpod/boltdb_state.go

```diff
@@ -413,7 +413,7 @@ func (s *BoltState) ValidateDBConfig(runtime *Runtime) error {
 		return err
 	}
 
-	return nil
+	return err
```

Comment:
Why is this changed, if anything this should just be `return checkRuntimeConfig(db, runtime)`